packages/synapse-react-client/src/synapse-client/SynapseClient.ts

```diff
@@ -2074,6 +2075,8 @@ export const uploadFile = (
   contentType: string = file.type,
   progressCallback?: (progress: ProgressCallback) => void,
   getIsCancelled?: () => boolean,
+  onMd5Computed?: () => void,
```

Comment:
Add `onMD5Computed` so we can tell the user when a file is finished 'processing' and starts being uploaded